test/integration/emulator/SfuClient.h

```diff
@@ -702,7 +705,7 @@ class SfuClient : public transport::DataReceiver
                         rtpHeader->ssrc.get(),
                         rtpMap,
                         sender,
-                        _audioType,
+                        _expectedReceiveAudioType == Audio::None ? _audioType : _expectedReceiveAudioType,
```

Comment:
??

Reply:
For backward compatibilty - if setter is not called, we'll expect _audioType. Otherwise - new _expectedREceiveAudioType